docs/ml-classification-regression.md

```diff
@@ -563,6 +563,15 @@ Refer to the [Python API docs](api/python/pyspark.ml.html#pyspark.ml.classificat
 {% include_example python/ml/fm_classifier_example.py %}
 </div>
 
+<div data-lang="r" markdown="1">
+
+Refer to the [R API docs](api/R/spark.fmClassifier.html) for more details.
+
+Note: At the moment SparkR doesn't suport feature scaling.
+
+{% include_example r/ml/fmClassifier.R %}
```

Comment:
Thanks for the quick update.

Actually, this should be ```{% include_example classification r/ml/fmClassifier.R %}``` or remove the ```classification``` from these two lines in the example (It might be better to update the example):
```# $example on:classification$```
```# $example off:classification$```
Otherwise, the docs build will fail. 

When I do documentation changes, I normally build and check. Here are the instructions for how to build the docs:
https://github.com/apache/spark/tree/master/docs

Reply:
Yeah, let's put it on hold for a few weeks, until I am on environment on which I can test things.